app/src/main/assets/special_thanks.csv

```diff
@@ -1,4 +1,6 @@
-221 Pixels;Logo design;https://avatars2.githubusercontent.com/u/58243143?s=60&v=4
-Anxhelo Lushka;Website design;https://avatars2.githubusercontent.com/u/25004151?s=60&v=4
-ByteHamster;Forum admin;https://avatars2.githubusercontent.com/u/5811634?s=60&v=4
-Keunes;Communications;https://avatars2.githubusercontent.com/u/11229646?s=60&v=4
+ByteHamster;Project lead;https://avatars2.githubusercontent.com/u/5811634?s=60&v=4
+Keunes;Project lead;https://avatars2.githubusercontent.com/u/11229646?s=60&v=4
+Femmdi;Translations coordinator;https://avatars2.githubusercontent.com/u/47671383?s=60&v=4
+Ryan Gorley (Freehive);Brand design;https://avatars2.githubusercontent.com/u/12849958?s=60&v=4
+221 Pixels;Initial logo design;https://avatars2.githubusercontent.com/u/58243143?s=60&v=4
```

Comment:
How about `Brand design (AntennaPod 2.x)` and `Brand design (AntennaPod 3.x)`

Reply:
That's not entirely true. The logo design stays the same, so it's not like the 2.0 design is now 'gone' (it's an iteration). Also, Ryan is doing more for us than just the icon.

What I could imagine, is that we put 'Logo update & brand design' for Ryan. (But then it becomes longer than the other entries.)

Reply:
> Also, Ryan is doing more for us than just the icon.

221 Pixels also did more than just the icon (Logo, wordmark, website blobs, color scheme, teaser image). Not nearly as throughout as Ryan, but they did something